UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/ViewComponents/BannerNotificationViewComponent.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.Web.ViewComponents
+{
+    [ViewComponent(Name = "BannerNotification")]
+    public class BannerNotificationViewComponent : ViewComponent
+    {
+        private readonly IMSIBannerNotificationService _mSIBannerNotificationService;
+
+        public BannerNotificationViewComponent(IMSIBannerNotificationService mSIBannerNotificationService)
+        {
+            _mSIBannerNotificationService = mSIBannerNotificationService;
+        }
+
+        public async Task<IViewComponentResult> InvokeAsync()
+        {
+            ViewBag.BannerNotificationMessage = await _mSIBannerNotificationService.GetBannerNotification();
+
```

Comment:
Please add logs and try/catch to log and throw error in case anything goes wrong.

Reply:
done required changes.